keyvault/azure-securtiy-keyvault-jca/run-with-command-line-client-side/README.md

```diff
@@ -0,0 +1,72 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- azure-key-vault
+description: "Sample for running executable jar with Azure key vault"
+urlFragment: "azure-spring-boot-sample-run-with-command-line-client-side"
+---
+
+# Run Key-vault-agnostic Web Client Executable Jar with Azure Key Vault JCA via Command Line 
+
+## Key concepts
+This sample illustrates how to run a web client with Azure key vault jca via command line.
```

Comment:
I think only one README.md is better, since client and server work together

Reply:
The server can work alone. However, the client depends on the server.